### PR TITLE
Adds ComposerPostProcess with example

### PR DESCRIPTION
**Pull request by pdagobert**

### CODE REVIEW

File: examples/new-postprocess/main.js [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+            this._viewer.init();
+            this._viewer.setSceneData(this._createScene());
+            this._viewer.setupManipulator();
+            this._viewer.getManipulator().computeHomePosition();
+            this._viewer.run();
+
+            this._rebuildGUI('passthrough.json');
+        },
+
+        _createCamera: function(width, height, texture) {
```
Comment (jtorresfabra) [inline examples/new-postprocess/main.js]: we don't need all the methods to be private. Just for coherence with other examples

File: examples/postprocess/bloom.js [hunk 2]
```diff
@@ -1,179 +0,0 @@
-(function() {
```
Comment (jtorresfabra) [inline examples/postprocess/bloom.js]: If you delete postprocess example it does not make sense to have the new one as new-postprocess does it?

File: sources/osgUtil/ComposerPostProcess.js [hunk 3]
```diff
@@ -0,0 +1,1346 @@
...
+                this._setInferedParameter('srgbInfered', 'srgb', outTexture, texture);
+            }
+
+            if (lastPassInfo.srgb === undefined) {
+                lastPassInfo.srgb =
+                    outTexture.srgb === undefined ? outTexture.srgbInfered : outTexture.srgb;
+            }
+        },
+
+        _setInferedParameter: function(paramInfer, param, dest, src) {
```
Comment (jtorresfabra) [inline sources/osgUtil/ComposerPostProcess.js]: mean inferred?

File: sources/osgUtil/ComposerPostProcess.js [hunk 4]
```diff
@@ -0,0 +1,1346 @@
...
+                    pass.out.name = passName;
+                }
+
+                outPasses.push(pass);
+            }
+
+            return outPasses;
+        },
+
+        // the number of textures needed by the post-processing stask can be deduced using the user data
```
Comment (jtorresfabra) [inline sources/osgUtil/ComposerPostProcess.js]: typo stask

**Comment by stephomi** [comment]
Fine for merge but I'm improving stuff in https://github.com/cedricpinson/osgjs/pull/862

**Pull request merged**